onprc_ehr/resources/views/PMIC_Wiki.html

```diff
@@ -0,0 +1,47 @@
+<p><strong>This page contains the links related to the PMIC data entry services.</strong></p>
+<p>&nbsp;</p>
+<ul>
+    <li>
+        <h4>Click here to go to,&nbsp;<strong><a href="/labkey/ehr/ONPRC/EHR/dataEntryForm.view?formType=PMIC%20SERVICES%20REQUEST">PMIC Services Request</a></strong>&nbsp;form</h4>
```

Comment:
Unfortunately this and the other links won't work on the production and other servers. Your dev machine is using the /labkey context path, but those servers are deployed directly in the root.

```suggestion
        <h4>Click here to go to,&nbsp;<strong><a href="<%=contextPath%>/ehr/ONPRC/EHR/dataEntryForm.view?formType=PMIC%20SERVICES%20REQUEST">PMIC Services Request</a></strong>&nbsp;form</h4>
```

Reply:
Thanks Josh. I have looked at the links in production and couldn't find any "contextpath" variable within the links for any servers. The links start with /ehr....
I will make that change and re-submit the files.